### PR TITLE
Helpertext

### PR DESCRIPTION
**Pull request by RachellCalhoun**
added helper text to model for ourstory. Also changed order = charfield so I can save.order as a title in admin page. Also added customization in admin as well to show order/color?

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
@hassanabidpk or @jinpark can you check it out?

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-coverage==4.0.3
```
Comment (hassanabidpk) [inline requirements.txt:1]: I added this along with test :smile:

Comment (RachellCalhoun) [inline requirements.txt:1]: ok i'll add it back. do i need to download it?

File: content/admin.py [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+
+class MCEFlatPageAdmin(FlatPageAdmin):
+    """
+    Page Admin
+    """
+    form = MCEFlatPageForm
+
+
+admin.site.unregister(FlatPage)
+admin.site.register(FlatPage, MCEFlatPageAdmin)
```
Comment (jinpark) [inline content/admin.py]: all this and the settings doesnt have anything to do with helper text, right?

Can you redo this PR with only the helper text stuff?

**Comment by RachellCalhoun** [comment]
i think i did everything @jinpark asked. let me know if i missed something.

File: content/migrations/0016_auto_20160320_0836.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import tinymce.models
```
Comment (jinpark) [inline content/migrations/0016_auto_20160320_0836.py:5]: this seems to error for me when running migrations

Comment (jinpark) [inline content/migrations/0016_auto_20160320_0836.py:5]: this SHOULD work now after you merge in master

**Comment by RachellCalhoun** [comment]
So, merge? or is there something else I should do?

**Comment by jinpark** [comment]
still reviewing :D

I'll comment if it seems fine

**Comment by jinpark** [comment]
Oh yes, you should merge master INTO this branch.

**Comment by RachellCalhoun** [comment]
git checkout helpertext
git rebase master  ? or just git merge master and it's not master because it's  fork. SO...i don't know.

**Comment by jinpark** [comment]
If you forked from the djangogirlsseoul repo

`git fetch upstream`
`git checkout master`
`git merge upstream/master`
`git checkout helpertext`
`git merge master`

Copied from https://help.github.com/articles/syncing-a-fork/

**Comment by jinpark** [comment]
I'm not a big fan of rebasing. I would suggest reading https://www.atlassian.com/git/tutorials/merging-vs-rebasing/workflow-walkthrough on why I don't think we should do rebase on our public projects

**Comment by RachellCalhoun** [comment]
I did something, it didn't work. I'm lost.

**Comment by jinpark** [comment]
Lets hold off on this PR until next time we meet

**Comment by RachellCalhoun** [comment]
Well, all my PRs are kinda weird now. My master is correct but my branch won't merge, it says its up to date with master but i can't see the tinymce changes reflected in the helpertext branch.

**Comment by jinpark** [comment]
why would all your PRs be weird? Looks like all the other ones are fine.

This one undos the mce PR you had before probably due to incorrect merging

**Pull request closed without merging**